readthedocs/restapi/views/model_views.py

```diff
@@ -205,6 +205,13 @@ class BuildViewSetBase(UserSelectViewSet):
     admin_serializer_class = BuildAdminSerializer
     model = Build
 
+    def get_queryset(self):
```

Comment:
Can you call `super` instead of making the `queryset`?
Something like
```python
queryset = super(self, BuildViewSetBase).get_queryset()
commit = self.request.query_params.get('commit', None)
if commit is not None:
    query = query.filter(commit=commit)
    return query
```

Reply:
I understand and I have looked into it as to why you asked me to do so as well. Your assistance was very well appreciated and I have made the requested changes. You can review it if you wish @safwanrahman.